server/middleware/__tests__/validation.spec.js

```diff
@@ -24,6 +24,21 @@ describe('Validators', () => {
         });
     });
 
+    describe('Numeric validator', () => {
+        it('should reject symbols', () => {
+            expect(validators.numeric({ value: '!@£$' })).not.toEqual(null);
+        });
+        it('should reject alpha', () => {
+            expect(validators.numeric({ value: 'data' })).not.toEqual(null);
+        });
+        it('should reject alphanumeric', () => {
+            expect(validators.numeric({ value: 'a1b2' })).not.toEqual(null);
+        });
+        it('should accept numeric only', () => {
+            expect(validators.numeric({ value: '1234' })).toEqual(null);
+        });
+    });
```

Comment:
you could also add tests to cover the message/label logic

Reply:
Additional coverage added as suggested. I'll do the same to the alphanumeric validator, too.